gcc/d/patches/patch-versym-os-4.9.x

```diff
@@ -250,19 +250,22 @@ These official OS versions are not implemented:
  #else
 --- gcc/config/linux.h
 +++ gcc/config/linux.h
-@@ -49,6 +49,20 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
+@@ -49,6 +49,23 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
  	builtin_assert ("system=posix");			\
      } while (0)
  
 +#define TARGET_OS_D_BUILTINS() TARGET_GENERIC_LINUX_OS_D_BUILTINS()
 +#define TARGET_GENERIC_LINUX_OS_D_BUILTINS()			\
 +    do {							\
-+	if (OPTION_GLIBC)					\
++	if (OPTION_GLIBC) {					\
 +	  builtin_define ("GNU_GLibc");				\
-+	else if (OPTION_UCLIBC)					\
++	  builtin_define ("CRuntime_GLibc");			\
++	} else if (OPTION_UCLIBC) {				\
 +	  builtin_define ("GNU_UCLibc");			\
-+	else if (OPTION_BIONIC)					\
++	} else if (OPTION_BIONIC) {				\
 +	  builtin_define ("GNU_Bionic");			\
++	  builtin_define ("CRuntime_Bionic");			\
++	}							\
```

Comment:
Whoops, was following the `do {` style; missed that it used the GNU stuff for actual code.